contracts/strategies/PlatypusVoter.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.3;
+
+import "../interfaces/IWAVAX.sol";
+import "../interfaces/IVePTP.sol";
+import "../lib/SafeERC20.sol";
+import "../lib/Ownable.sol";
+import "../lib/ERC20.sol";
+
+contract PlatypusVoter is Ownable, ERC20 {
```

Comment:
may make sense to use with YakERC20 instead (https://github.com/yieldyak/smart-contracts/blob/master/contracts/YakERC20.sol)

Reply:
I looked at YakERC20, but the _burn function isn't good for this and you can't change the name and symbol right now.